app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,77 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserData;
+import com.codesoom.assignment.exception.UserNotFoundException;
+import com.codesoom.assignment.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    /**
+     * id 값과 일치하는 User를 리턴합니다
```

Comment:
JavaDoc의 메인 설명문은 많은 사람들에게 보여주기 위한 문서화가 되는 문장입니다. 마침표를 찍어주세요.
```suggestion
     * id 값과 일치하는 User를 리턴합니다.
```